mautic/mautic
# TPROD-437 Removing unused PlaySound JS library

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ cleanup ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://mautic.atlassian.net/browse/TPROD-437



#### Description:

The playsound JS library was added in https://github.com/mautic/mautic/commit/9e1f804cfec0e218ee82ffbb86be9bfe3f2e71ca  for the Chat Bundle that was probably deleted a long time ago. It’s not used and can be removed.

https://github.com/admsev/jquery-play-sound 

https://github.com/mautic/mautic/blob/5.x/app/bundles/CoreBundle/Assets/js/libraries/11.jquery.playsound.js

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Load the Mautic administration and check that there are no JS errors.
